TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -1144,10 +1148,10 @@ public Token GetNextToken()
                             }
 
                             delimiterToken = new Token(TokenType.PseudoTextDelimiter, startIndex, startIndex + 1, usesVirtualSpaceAtEndOfLine, tokensLine);
-                            if (!(followingChar == ' ' || followingChar == ',' || followingChar == ';' || followingChar == '.'))
-                            {
-                                tokensLine.AddDiagnostic(MessageCode.InvalidCharAfterPseudoTextDelimiter, delimiterToken);
-                            }
+                            //if (!(followingChar == ' ' || followingChar == ',' || followingChar == ';' || followingChar == '.'))
+                            //{
+                            //    tokensLine.AddDiagnostic(MessageCode.InvalidCharAfterPseudoTextDelimiter, delimiterToken);
+                            //}
```

Comment:
Commented code may be confusing for future readers, better remove it.

Reply:
So this code is commented because, I still needed to know how to emit this diagnostics in  a right context.